client/dashboard/utils/site-url.ts

```diff
@@ -1,8 +1,8 @@
 import type { Site } from '../data/types';
 
-export function getSiteHostname( site: Site ) {
+export function getSiteDisplayUrl( site: Site ) {
 	if ( site.URL === '' ) {
 		return site.URL;
 	}
```

Comment:
Now that we're not using `new URL` we can't crash on empty strings anymore. This whole change is unnecessary.